ios/RNViewShot.m

```diff
@@ -19,6 +19,12 @@ - (dispatch_queue_t)methodQueue
   return RCTGetUIManagerQueue();
 }
 
+RCT_EXPORT_METHOD(captureScreenshot: (NSDictionary *)options
+                  resolve:(RCTPromiseResolveBlock)resolve
+                  reject:(RCTPromiseRejectBlock)reject) 
+{
+  [captureRef -1, options, resolve, reject];
```

Comment:
I think the code should be `[self.captureRef -1 withOptions:options resolver:resolve reject:reject]`

<img width="761" alt="screen shot 2017-09-16 at 09 34 51" src="https://user-images.githubusercontent.com/211411/30510360-725b00fa-9ac2-11e7-9ccb-fef6f85f7174.png">

Reply:
Error resolved.